community/working-groups.md

```diff
@@ -0,0 +1,44 @@
+## Working Groups
+
+A working group is a group of people that are interested in working on a common
+topic. Working groups are informal and have no official authority. All working
+groups have a public channel on the [Julia Slack][slack]. Many working groups
+also hold regular meetings that are held online and are open to anyone. Meeting
+links are posted in the relevant Slack channels.
+
+The working group meetings are also listed on the
+[community events calendar][commmunity-events-calendar].
+
+### List of working groups
+
+Note: "ET" refers to Eastern Time.
+
+| Working Group                    | Slack channel     | Meetings                                 | Contact                                                   |
+| -------------------------------- | ----------------- | ---------------------------------------- | --------------------------------------------------------- |
+| Base Julia CI                    | `#ci-dev`         | Every other Tuesday at 2:15pm ET         | Elliot Saba (@staticfloat), Dilum Aluthge (@DilumAluthge) |
+| GPU                              | `#gpu`            | Every other Monday at 11am ET            | Tim Besard (@maleadt), Valentin Churavy (@vchuravy) |
+| High-performance computing (HPC) | `#hpc`            | Fourth Tuesday of each month at 2pm ET   | Valentin Churavy |
+| Multithreading                   | `#multithreading` | First Wednesday of each month at 10am ET | Jameson Nash (@vtjnash)|
+| Pkg                              | `#pkg-dev`        | Occasional Tuesdays at 2:15pm ET         | Kristoffer Carlsson (@KristofferC), Fredrik Ekre (@fredrikekre) |
+| Release management               | `#releases`       | [none]                                   | Kristoffer Carlsson, Alex Arslan (@ararslan)|
```

Comment:
Add the machine learning call, even though it's mostly in the FluxML org?

Reply:
Sure! Which ML call is it? There's a couple different calls on the community calendar that have "ML" in the name.

Reply:
And who would be a good person (or two people) to list in the "Contact" column?

Reply:
IIRC @darsnack hosts the meeting or if not he'll be able to say.

Reply:
Yeah I usually host but you can list @lorenzoh too

Reply:
Officially, we use Zulip to coordinate but usually cross post to `#machine-learning` or `#flux-bridged` if you need a Slack channel.

Reply:
What's the schedule for the meetings?

Also, what name should we use to describe said working group?

Reply:
Oh, and what's the Slack channel?

Reply:
Every other Tuesday starting today

Reply:
I've put `#machine-learning` as the Slack channel, but if there's a different channel that would be better, we can change it!